File: docs/versioned_docs/version-1.11.0/API-Reference/javascript-examples/api-projects/create-project-2.js
const url = `${process.env.LANGFLOW_URL ?? ""}/api/v1/projects/`;

const options = {
  method: 'POST',
  headers: {
    "accept": `application/json`,
    "Content-Type": `application/json`,
    "x-api-key": `${process.env.LANGFLOW_API_KEY ?? ""}`,
  },
  body: JSON.stringify({
  "name": "new_project_name",
  "description": "string",
  "components_list": [
    "3fa85f64-5717-4562-b3fc-2c963f66afa6"
  ],
  "flows_list": [
    "3fa85f64-5717-4562-b3fc-2c963f66afa6"
  ]
}),
};

fetch(url, options)
  .then(async (response) => {
    if (!response.ok) {
      throw new Error(`HTTP ${response.status}`);
    }
    const text = await response.text();
    console.log(text);
  })
  .catch((error) => console.error(error));

File: docs/versioned_docs/version-1.12.0/API-Reference/python-examples/api-projects/create-project-2.py
import os

import requests

url = f"{os.getenv('LANGFLOW_URL', '')}/api/v1/projects/"

headers = {
    "accept": "application/json",
    "Content-Type": "application/json",
    "x-api-key": f"{os.getenv('LANGFLOW_API_KEY', '')}",
}

payload = {
    "name": "new_project_name",
    "description": "string",
    "components_list": ["3fa85f64-5717-4562-b3fc-2c963f66afa6"],
    "flows_list": ["3fa85f64-5717-4562-b3fc-2c963f66afa6"],
}

response = requests.request("POST", url, headers=headers, json=payload)
response.raise_for_status()

print(response.text)
